code/modules/client/client_procs.dm

```diff
@@ -1007,6 +1007,5 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 		if(!istext(verb_to_init:category))
 			continue
 		verb_tabs |= verb_to_init:category
-		verblist[++verblist.len] = list("[verb_to_init:category]", "[verb_to_init:name]")
+		verblist[++verblist.len] = list(verb_to_init:category, verb_to_init:name)
```

Comment:
One last thing before merge, can you make sure everything that does this instead uses `/procpath`, as mentioned in another review?

It's a fake static type that lets us do normal indexing.

```
var/procpath/verb_to_init = whatever
do_somehting(verb_to_init.category, verb_to_init.name)
```

Reply:
done